packages/terra-list/src/ListSectionHeader.jsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import 'terra-base/lib/baseStyles';
+import ListUtils from './ListUtils';
+import styles from './ListSectionHeader.module.scss';
+
+const cx = classNames.bind(styles);
+
+/* eslint-disable react/forbid-prop-types */
+const propTypes = {
+  /**
+   * Whether or not the section is collapsed.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Whether or not the section can be collapsed.
+   */
+  isCollapsible: PropTypes.bool,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`. Default `level=2`.
```

Comment:
Do we need to specify default as part of the comment? Our props table does come with a defaultValue column

Reply:
Updated.